Lesson7/deniz/index.html

```diff
@@ -1,11 +1,211 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
-  <meta charset="UTF-8">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="resp-design.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.5.1/css/all.min.css">
+    <title>Medium2</title>
 </head>
 <body>
+    <header>
+
+    </header>
+
+    <main>
+        
+        
+        <div class="container-top">
+
+          <div class=content-one>  
+                <h1 class="h1-top">Time-saving CSS techniques to create responsive images</h1>
+            
+                <div class="profile-info">
+                       <div class="profil-photo">
+                            <img class="small-pic" src="./img/IMG_7036.png" alt="Adrien Zaganelli"> 
+                            </div>
+            
+                            <div class="top-div">
+                            <div class="Adrien-link">
+                            <a href="#">Adrien Zaganelli</a>
+                            <a href="#"><p class="fbtn">Follow</p></a>
```

Comment:
No need to use <p> inside a link
```suggestion
                            <a href="#" class="fbtn">Follow</a>
```